resources/assets/components/CampaignSingle/index.js

```diff
@@ -55,6 +55,26 @@ class CampaignSingle extends React.Component {
             label: 'Bulk',
             active: false,
           },
+          irrelevant: {
+            label: 'Irrelevant',
+            active: false,
+          },
+          inappropriate: {
+            label: 'Inappropriate',
+            active: false,
+          },
+          'unrealistic-quantity': {
```

Comment:
this is weird to me but I think this is part of the automatic styling to have `'` when there is more than one word in the tag! It wouldn't let me add `'` for the single words!

Reply:
this is strange!!

Reply:
Yep, this is how Prettier formats objects (no quotes unless necessary). In most cases, we use "plain" words for keys (like `label` and `active` above) and so quotes are just noise, but `unrealistic-quantity` would be a syntax error without the quotes.